packages/btcindexer/src/btcindexer.ts

```diff
@@ -401,4 +428,57 @@ export class Indexer implements Storage {
 		}
 		return updates;
 	}
+
+	async getStatusByTxid(txid: string): Promise<TransactionStatusResponse | null> {
+		const latestHeightStr = await this.nbtcTxDB.get("chain_tip");
```

Comment:
chain_tip should be stored in `blocksDB`, not in nbtcTXDB

Reply:
done